src/main/java/com/github/mc1arke/sonarqube/plugin/ce/pullrequest/azuredevops/model/enums/CommentThreadStatus.java

```diff
@@ -0,0 +1,36 @@
+package com.github.mc1arke.sonarqube.plugin.ce.pullrequest.azuredevops.model.enums;
+
+/**
+ * The status of a comment thread
+ */
+public enum CommentThreadStatus {
+    /**
+     * The thread status is unknown.
+     */
+    unknown,
```

Comment:
Naming conventions are to have enum fields in upper-case. Is there any reason that can't be followed here?

Reply:
Resolve

Reply:
Resolved